Sources/PackageDescription/PackageDescription.swift

```diff
@@ -22,7 +22,7 @@ import Foundation
 
 /// The configuration of a Swift package.
 ///
-/// Pass configuration options as parameters to your package's initializer
+/// Pass configuration options as parameters to your package’s initializer
```

Comment:
Although it's correct (page 270 of ) to use U+2019 instead of U+0027 for punctuation, it can be more difficult to edit it using many IDE and keyboard combinations. (see page 270 of [Unicode 13.0 spec](https://www.unicode.org/versions/Unicode13.0.0/ch06.pdf) which outlines the dilemma)

Reply:
This was due to my internal process, and I thank you for calling it out.

Reply:
There are other locations where U+0027 is changed to U+2019. If the changes are not intended, then they should be changed back, too.

Reply:
I changed all of them to the straight apostrophe under `PackageDescription`.